src/cmd/linuxkit/pkglib/build.go

```diff
@@ -361,7 +361,7 @@ func (p Pkg) buildArch(d dockerRunner, c lktspec.CacheProvider, arch string, arg
 		tag     = p.Tag()
 	)
 	switch arch {
-	case "amd64", "arm64", "s390x":
+	case "amd64", "arm64", "s390x", "riscv64":
```

Comment:
This should be the one piece necessary to enable building on `riscv64`. Personally, I would like to see it gone. If someone explicitly selected to build for arch `foobar128`, then `buildArch` should not be filtering it out. I don't think we should have the restricted set of archs hard-coded into `buildArch()`.

I am going to propose that you just remove this entire `switch arch` clause, replacing it with just `tagArch = tag+"-"+arch` that is in the successful case.

Reply:
Done.